source/_patterns/05-pages/autospacing/autospacing.twig

```diff
@@ -0,0 +1,80 @@
+{% include '@organisms/global/header/header.twig' %}
+{% include '@molecules/blocks/hero/hero.twig'  with {
+    title: top.hero.title,
+    lead: top.hero.lead,
+    body: top.hero.body,
+    column_variant: top.hero.column_variant,
+    column_content_right: top.hero.column_content_right,
+    breadcrumbs: top.breadcrumbs
+} %}
+
+{# Default Body Section #}
+{% for entry in entries %}
+    <section class="{{ entry.background_variant }}{% if entry.background_variant_token %}--{{ entry.background_variant_token }}{% endif %}">
```

Comment:
Why are we wrapping these in another element?  Wasn't the idea to just use the background variant classes that are already there?

Reply:
Ah, ok that makes sense now. Thanks!